apps/server/package.json

```diff
@@ -13,7 +13,17 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "ts-node-dev": "^2.0.0"
+    "@typescript-eslint/eslint-plugin": "^6.7.0",
+    "@typescript-eslint/parser": "^6.7.0",
+    "eslint": "^8.49.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-config-standard-with-typescript": "^39.0.0",
```

Comment:
@celikmurat1845, it seems you are not using plugins at line 20,21,22,23. I am confused, if you want to use them you should add them to your config file, else please remove them.